HW_2/task_2.py

```diff
@@ -0,0 +1,18 @@
+#"""
+#Напишите программу, которая принимает на вход натуральное число N и выдает список факториалов по основаниям от 1 до N
+
+#Ввод: значение типа <int>
+#Вывод: значение типа <list>
+
+#Пример:
+#4
+#[1, 2, 6, 24]
+#"""
+n = int(input("Введите число n: "))
+my_list = []
+f = 1
+for i in range (1, n+1):
+        f = f * i
+        my_list.append(f)
+        print(my_list)
```

Comment:
Зачем же это выводить каждую итерацию цикла?
```suggestion
print(my_list)
```